Santouka/css/main.css

```diff
@@ -0,0 +1,197 @@
+/* 共通 */
+body {
+  margin: 0;
+  background: url(../img/background2.jpeg) center center no-repeat;
+  width: 100%;
+  height: 52rem;
+  background-size: cover;
+  box-sizing: border-box;
+  background-color:rgba(255,255,255,0.5);
+  background-blend-mode: lighten;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/*--------------------
+header
+--------------------*/
+header {
+  /* border: 2px solid red;  */
+  background-color:rgb(46, 45, 45);
+  display: flex;
+}
+
+header img {
+  /* border: 2px solid blue;  */
+  height: 4em;
+}
+
```

Comment:
maybe you should start to use class using approach.
html tags for styling is weak for html changes.

ex.
change tags from h1 to div, then all style will gone....

Reply:
style and tags should be independent

Reply:
I fixed!